RePoE/parser/util.py

```diff
@@ -80,3 +80,10 @@ def ignore_mod_domain(domain):
         MOD_DOMAIN.AFFLICTION_JEWEL,
     }
     return domain not in whitelist
+
+
+def find_missing_stat_descriptions(file_system, data_path, relational_reader, translation_file_cache, ot_file_cache):
```

Comment:
Just a tiny thing: Since this isn't called using `call_with_default_args` anymore, it only needs the first parameter.